nncf/graph/operator_metatypes.py

```diff
@@ -156,13 +156,13 @@ def get_operator_metatype_by_op_name(self, op_name: str) -> 'OperatorMetatype':
 @OPERATOR_METATYPES.register()
 class InputNoopMetatype(OperatorMetatype):
     name = "input_noop"
-    external_op_names = [name, MODEL_INPUT_OP_NAME]
+    external_op_names = [name, MODEL_INPUT_OP_NAME, NNCFGraphNodeType.INPUT_NODE]
```

Comment:
Same here - what is the meaning of this, seeing that NNCFGraphNodeType is an integer-valued enum?